src/request.ts

```diff
@@ -118,6 +118,7 @@ async function startFetchRequest(
         ? { body: input }
         : {}),
       redirect: 'manual',
+      // @ts-expect-error - differing types of signal? this started abruptly
```

Comment:
yep, we use an AbortController polyfill:
https://github.com/mysticatea/abort-controller/issues/36

might drop it in v4 (use browser/node native AC).